src/settings.rs

```diff
@@ -355,35 +356,51 @@ impl Settings {
         settings
     }
 
-    // Use ~/.mcfly only if it already exists, otherwise create 'mcfly' folder in XDG_CACHE_DIR
+    // Use or create the 'mcfly' folder in `$XDG_CACHE_DIR``
     pub fn mcfly_training_cache_path() -> PathBuf {
-        let cache_dir = Settings::mcfly_xdg_dir().cache_dir().to_path_buf();
-
-        Settings::mcfly_base_path(cache_dir).join(PathBuf::from("training-cache.v1.csv"))
+        Settings::xdg_base_strategy()
```

Comment:
What is the difference between mcfly_xdg_dir and xdg_base_strategy?

Reply:
I could reuse the name if you want.

Reply:
I just didn't understand what the difference is between them.

Reply:
`mcfly_xdg_dir` returned a `ProjectDirs` from the `directories` crate. `xdg_base_strategy` base directory returns an `Xdg` struct from `etcetera` crate which is the counterpart to it.